requirements/docs.txt

```diff
@@ -9,6 +9,11 @@ sphinx-rtd-theme==0.2.4
 # Needed to build markdown docs
 recommonmark==0.4.0
 
+# Needed for documentation_crawler
+pathlib2==2.2.1
+#Needed for pathlib2==2.2.1
+scandir==1.5
+
```

Comment:
these should not be moved here; `docs.txt` is used just for ReadTheDocs to build the documentation; not for testing it.